omod/src/main/resources/liquibase.xml

```diff
@@ -4,6 +4,12 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
 
+	<changeSet id="BAH-828-001" author="Samuel34">
```

Comment:
Can use a convention of "BACT-YYYYMMDDHHMM"  (something that we are trying to enforce across the different modules) for changeset ID, use "author" field, also use "context" field to mention "release/version" applicable. You can use Comment to mention the specific JIRA card as well.